### PR TITLE
vnstati: add page

### PR DESCRIPTION
**Pull request by melvin2204**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).

### CODE REVIEW

**Comment by CleanMachine1** [comment]
Please convert this into a ready PR

File: pages/linux/vnstati.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# vnstati
+
+> png image output support for vnStat
```
Comment (CleanMachine1) [inline pages/linux/vnstati.md:3]:
```suggestion
> PNG image output support for vnStat.
```

Comment (melvin2204) [reply pages/linux/vnstati.md]: Changed and pushed.

**Comment by CleanMachine1** [comment]
Could you provide a `More information: link.`

File: pages/linux/vnstati.md [hunk 2]
```diff
@@ -1,6 +1,7 @@
 # vnstati
 
 > PNG image output support for vnStat.
+> More information: <https://linux.die.net/man/1/vnstati>.
```
Comment (CleanMachine1) [inline pages/linux/vnstati.md:4]:
```suggestion
> More information: <https://manned.org/vnstati.1>.
```

We use manned.org when using man pages

**Comment by CleanMachine1** [comment]
You must sign the license agreement. Hope you aren't having a problem doing so

File: pages/linux/vnstati.md (hunk 2 above)
Comment (melvin2204) [reply pages/linux/vnstati.md]: Never heard of manned.org before, but I updated it.

**Comment by CleanMachine1** [comment]
Also why aren't you just using the suggestions?

**Comment by CleanMachine1** [comment]
Does this command have long flags for example `-o to --output`?

**Comment by melvin2204** [comment]
> 
> 
> Also why aren't you just using the suggestions?

I'm sorry but what do you mean with the suggestions? This is my first time contributing to projects that aren't my own, so I'm trying to figure everything out.

**Comment by CleanMachine1** [comment]
Yes

**Comment by CleanMachine1** [comment]
On suggestions, you can press `commit suggestion` then apply it as a commit

**Comment by melvin2204** [comment]
I found the styling guide and reworked the styling. I followed the styling from [pages/linux/vnstat.md](vnstat) so that one might need to be reworked too.

**Comment by CleanMachine1** [comment]
I don't like *.png 

output was better

**Comment by CleanMachine1** [comment]
Maybe I'll open a PR to change that and good thinking using the previous page, thats how I like to make mine since there is a lot less reviewing to done

**Comment by melvin2204** [comment]
> 
> 
> I don't like *.png
> 
> output was better

Changed it back to {{output.png}}.

**Comment by CleanMachine1** [comment]
Also have you made sure the flags are valid? Did you check or just assume and copy, I've made the mistake before

**Comment by melvin2204** [comment]
> 
> 
> Also have you made sure the flags are valid? Did you check or just assume and copy, I've made the mistake before

I tested them before opening the PR. Just tested it again and they still work.

File: pages/linux/vnstati.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+# vnstati
+
+> PNG image output support for vnStat.
+> More information: <https://manned.org/vnstati>.
+
+- Output summary of the last 2 days/months and all time:
```
Comment (CleanMachine1) [inline pages/linux/vnstati.md:6]:
```suggestion
- Output a summary of the last 2 months, days, and all time:
```

How is it chosen for whether it is days or months
if it is 1 or the other then `/` is wrong

Comment (melvin2204) [reply pages/linux/vnstati.md:6]: It shows the days and the months, so both. [Example output](https://i.imgur.com/exDJ0WI.png).

File: pages/linux/vnstati.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
+# vnstati
+
+> PNG image output support for vnStat.
+> More information: <https://manned.org/vnstati>.
+
+- Output a summary of the last 2: months, days, and all time:
```
Comment (SethFalco) [inline pages/linux/vnstati.md:6]:
```suggestion
- Output a summary of the last 2: months, days, and all-time:
```

* https://www.collinsdictionary.com/dictionary/english/all-time
* https://www.lexico.com/definition/all-time
* https://www.merriam-webster.com/dictionary/all-time

File: pages/linux/vnstati.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+
+`vnstati --summary --iface {{network_interface}} --output {{output.png}}`
+
+- Output all time top 10 traffic days:
```
Comment (SethFalco) [inline pages/linux/vnstati.md:10]:
```suggestion
- Output all-time top 10 traffic days:
```

File: pages/linux/vnstati.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+
+`vnstati --top10 --iface {{network_interface}} --output {{output.png}}`
+
+- Output traffic statistics on a monthly basis for the last 12 months:
+
+`vnstati --months --iface {{network_interface}} --output {{output.png}}`
+
+- Output traffic statistics on a hourly basis for the last 24 hours:
```
Comment (SethFalco) [inline pages/linux/vnstati.md:18]:
```suggestion
- Output traffic statistics on an hourly basis for the last 24 hours:
```

Should be "an" here, not "a"!

**Review by SethFalco — CHANGES_REQUESTED** [review]
Hey, and welcome to the repository! :wave:

Just a few comments regarding grammar!

File: pages/linux/vnstati.md (hunk 6 above)
Comment (melvin2204) [reply pages/linux/vnstati.md:18]: You are right, but I got that title from the man page, so they made the mistake, not me. 😅

**Review by SethFalco — APPROVED** [review]
I think it looks fine now. \\(^-^)/

File: pages/linux/vnstati.md [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+
+- Output a summary of the last 2: months, days, and all-time:
+
+`vnstati --summary --iface {{network_interface}} --output {{output.png}}`
+
+- Output all-time top 10 traffic days:
+
+`vnstati --top10 --iface {{network_interface}} --output {{output.png}}`
+
+- Output traffic statistics on a monthly basis for the last 12 months:
```
Comment (CleanMachine1) [inline pages/linux/vnstati.md:14]:
```suggestion
- Output traffic statistics for every month for the last 12 months:
```

Leave this suggestions open for others to see

File: pages/linux/vnstati.md [hunk 8]
```diff
@@ -0,0 +1,20 @@
...
+
+`vnstati --months --iface {{network_interface}} --output {{output.png}}`
+
+- Output traffic statistics on an hourly basis for the last 24 hours:
```
Comment (CleanMachine1) [inline pages/linux/vnstati.md:18]:
```suggestion
- Output traffic statistics for every hour for the last 24 hours:
```

File: pages/linux/vnstati.md [hunk 9]
```diff
@@ -0,0 +1,20 @@
+# vnstati
+
+> PNG image output support for vnStat.
+> More information: <https://manned.org/vnstati>.
+
+- Output a summary of the last 2: months, days, and all-time:
+
+`vnstati --summary --iface {{network_interface}} --output {{output.png}}`
+
+- Output all-time top 10 traffic days:
```
Comment (CleanMachine1) [inline pages/linux/vnstati.md:10]:
```suggestion
- Output all-time top 10 of network traffic days:
```

File: pages/linux/vnstati.md (hunk 8 above)
Comment (CleanMachine1) [reply pages/linux/vnstati.md:18]: ditto

File: pages/linux/vnstati.md (hunk 7 above)
Comment (CleanMachine1) [reply pages/linux/vnstati.md:14]:
```suggestion 

- Output traffic statistics for each month, for the last 12 months:

Comment (CleanMachine1) [reply pages/linux/vnstati.md:14]:
```suggestion 
- Output traffic statistics for every month, for the last 12 months:
```

File: pages/linux/vnstati.md [hunk 10]
```diff
@@ -0,0 +1,20 @@
+# vnstati
+
+> PNG image output support for vnStat.
+> More information: <https://manned.org/vnstati>.
+
+- Output a summary of the last 2: months, days, and all-time:
```
Comment (nicokosi) [inline pages/linux/vnstati.md:6]: `last 2`:  Why? Is there a default value?

Comment (nicokosi) [reply pages/linux/vnstati.md:6]: What does is mean?

Is is something like:
> Output a summary of the last 2 months, last 2 days, and all-time:

Comment (melvin2204) [reply pages/linux/vnstati.md:6]: > 
> 
> What does is mean?
> 
> Is it something like:
> 
> > Output a summary of the last 2 months, last 2 days, and all-time:

It shows the summary of today, yesterday, the current month, the previous month and the total usage. [This is the output.](https://i.imgur.com/exDJ0WI.png)

File: pages/linux/vnstati.md (hunk 7 above)
Comment (marchersimon) [reply pages/linux/vnstati.md:14]: Just another one:
```suggestion
- Output monthly traffic statistics from the last 12 months:
```

File: pages/linux/vnstati.md (hunk 8 above)
Comment (marchersimon) [reply pages/linux/vnstati.md:18]:
```suggestion
- Output hourly traffic statistics from the last 24 hours:
```

File: pages/linux/vnstati.md (hunk 9 above)
Comment (marchersimon) [reply pages/linux/vnstati.md:10]:
```suggestion
- Output the 10 most traffic-intensive days of all time:
```

**Comment by marchersimon** [comment]
@melvin2204 are you still working on this?

**Comment by melvin2204** [comment]
> 
> 
> @melvin2204 are you still working on this?

No, I think it is ready, but @CleanMachine1 said I needed to keep the suggestions open for others to see. So that is what I did.

**Comment by marchersimon** [comment]
Oh, I see. Sorry this PR got a bit lost in the shuffle.

File: pages/linux/vnstati.md (hunk 7 above)
Comment (marchersimon) [reply pages/linux/vnstati.md:14]: @melvin2204 just apply the one you like the most.

File: pages/linux/vnstati.md [hunk 11]
```diff
@@ -0,0 +1,20 @@
...
+
+`vnstati --summary --iface {{network_interface}} --output {{output.png}}`
```
Comment (CleanMachine1) [inline pages/linux/vnstati.md:8]:
```suggestion
`vnstati --summary --iface {{network_interface}} --output {{path/to/output.png}}`
```

File: pages/linux/vnstati.md [hunk 12]
```diff
@@ -0,0 +1,20 @@
...
+
+`vnstati --top10 --iface {{network_interface}} --output {{output.png}}`
+
+- Output monthly traffic statistics from the last 12 months:
+
+`vnstati --months --iface {{network_interface}} --output {{output.png}}`
+
+- Output hourly traffic statistics from the last 24 hours:
+
+`vnstati --hours --iface {{network_interface}} --output {{output.png}}`
```
Comment (CleanMachine1) [inline pages/linux/vnstati.md:20]:
```suggestion

`vnstati --summary --iface {{network_interface}} --output {{path/to/output.png}}`

- Output the 10 most traffic-intensive days of all time:

`vnstati --top10 --iface {{network_interface}} --output {{path/to/output.png}}`

- Output monthly traffic statistics from the last 12 months:

`vnstati --months --iface {{network_interface}} --output {{path/to/output.png}}`

- Output hourly traffic statistics from the last 24 hours:

`vnstati --hours --iface {{network_interface}} --output {{path/to/output.png}}`
```

**Review by CleanMachine1 — APPROVED** [review]
:shipit:

**Review by marchersimon — APPROVED** [review]
Thanks @melvin2204 and sorry for the long delay

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**